doc/antora/modules/reference/nav.adoc

```diff
@@ -118,14 +118,14 @@
 *** xref:man/radmin.adoc[radmin]
 *** xref:man/radsniff.adoc[radsniff]
 
-** xref:index.adoc[Configuration Files]
-*** xref:format.adoc[Format of the Configuration Files]
+** xref:raddb/index.adoc[Configuration Files]
+*** xref:raddb/format.adoc[Format of the Configuration Files]
 
-*** xref:certs/index.adoc[Certificates]
+*** xref:raddb/certs/index.adoc[Certificates]
 
-*** xref:global.d/index.adoc[Global Configuration]
-*** xref:global.d/ldap.adoc[ldap]
-*** xref:global.d/python.adoc[ldap]
+*** xref:raddb/global.d/index.adoc[Global Configuration]
+*** xref:raddb/global.d/ldap.adoc[ldap]
+*** xref:raddb/global.d/python.adoc[python]
```

Comment:
the rest of the links below this should change, too.  e.g. `mods-available/foo` -> `raddb/mods-available/foo`

drop this commit, and I'll push one which fixes all of that.